river/View.zig

```diff
@@ -65,6 +66,12 @@ const State = struct {
 
     float: bool = false,
     fullscreen: bool = false,
+
+    /// The opacity of this view
+    opacity: f32 = 0.0,
```

Comment:
Don't use a default value if initializing this to something else.
```suggestion
    opacity: f32,
```